File: 08-Testing-Java/Starter/src/testing/MyClass.java
package testing;

public class MyClass {
    public void doLoop() {
        System.out.println("Before loop");

        for (int i = 1; i <= 3; i++) {
            System.out.println("In loop: " + i);
        }

        System.out.println("After loop");
    }
}

File: 08-Testing-CSharp/Starter/Testing/MyClass.cs
namespace Testing;

public sealed class MyClass
{
    public void DoLoop()
    {
        Console.WriteLine("Before loop");

        for (var i = 1; i <=3; i++)
        {
            Console.WriteLine($"In loop: {i}");
        }

        Console.WriteLine("After loop");
    }
}